src/middlewares/user.ts

```diff
@@ -341,7 +341,13 @@ export const checkUserHasAtLeastOneOrganizationMiddleware = (
           ),
         )
       ) {
-        return res.redirect("/users/join-organization");
+        if (req.session.siretHint) {
```

Comment:
todo: note that this implementation, ignore the siret_hint params if `req.session.certificationDirigeantRequested === true`. This is not the expected behavior.